vnext/Microsoft.ReactNative/ReactInstance.idl

```diff
@@ -1,18 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-import "NoExceptionAttribute.idl";
-
-namespace Microsoft.ReactNative {
-
-  [webhosthidden]
-  interface IReactInstance {
-    void InvokeFunction(String moduleName, String method, IVectorView<IInspectable> arguments);
-  }
-
-  [webhosthidden]
-  [default_interface]
-  runtimeclass ReactInstance : IReactInstance {
```

Comment:
>ReactInstance [](start = 15, length = 13)

For my education: What takes its place?

Reply:
We do not expose the ReactInstance. Instead we offer the IReactContext.

---
In reply to: [431494732](https://github.com/microsoft/react-native-windows/pull/5030#discussion_r431494732) [](ancestors = 431494732)